src/app/Plans/components/Wizard/WizardComponent.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import WizardStepContainer from './WizardStepContainer';
+import { Wizard } from '@patternfly/react-core';
+import Review from './Review';
+import GeneralForm from './GeneralForm';
+import { Provider } from '@app/Providers/types';
+
+interface IWizardComponent {
+  isOpen: boolean;
+  onHandleWizardClose: () => void;
+  sourceProviders: Provider[];
+  targetProviders: Provider[];
+}
+
+const WizardComponent: React.FunctionComponent<IWizardComponent> = ({
```

Comment:
Maybe we should call this PlanWizard instead of WizardComponent, in case we eventually have another wizard? WizardComponent was only really necessary to distinguish from WizardContainer in mig-ui, which we don't need here. Totally not important, I'm fine either way.

Reply:
That's a good idea, changing it.